stepper.c

```diff
@@ -236,9 +232,19 @@ static uint16_t st_shutdown_delay;  //ms (max = 32767)
 //disable stepper output (0 to enable)
 void st_disable(uint8_t disable, uint8_t mask) {
   if (mask & sys.lock_mask) st_shutdown_start = 0;  //clear pending shutdown if we are enabling, or if it has pent.
-  if (bit_istrue(settings.flags,BITFLAG_INVERT_ST_ENABLE)) { disable = !disable; } // Apply pin invert.
-  if (disable) { STEPPERS_DISABLE_PORT |= (STEPPERS_DISABLE_MASK&mask); }
-  else { STEPPERS_DISABLE_PORT &= ~(STEPPERS_DISABLE_MASK&mask); }
+  if (bit_istrue(settings.flags, BITFLAG_INVERT_ST_ENABLE)) {
+    disable = !disable;  // Apply pin invert.
+  }
+
+  if (disable) {
+    STEPPERS_DISABLE_PORT |= (STEPPERS_DISABLE_MASK & mask);
+  } else { 
+    STEPPERS_DISABLE_PORT &= ~(STEPPERS_DISABLE_MASK & mask);
+  }
```

Comment:
Did anything in the disable logic change?  Mild preference to revert it if the logic is the same (or move it into its own commit at least).